File: java/lab3/Program.java
package lab3;

import java.util.Scanner;

class Program {
    public static void main(String[] args) {
        theLunchQueue();
    }

    public static int getInt(String prompt) {
        System.out.println(prompt);
        Scanner s = new Scanner(System.in);
        return s.nextInt();
    }

    public static String getString(String prompt) {
        System.out.println(prompt);
        Scanner s = new Scanner(System.in);
        return s.nextLine();
    }

    public static void theLunchQueue() {
        String mainCourse = getString("What main dish would you like(Fish, Burgers or veg) ?");
        int xx = getInt("How many roast potatos would you like?");
        int yy = getInt("How many Brussel Sprouts would you like?");
        System.out.printf("Hello, your lunch is %s with %d roast potatoes and %d Brussel sprouts", mainCourse, xx, yy);
    }
}

File: csharp/lab3/Program.cs
using System;

namespace Lab1
{
    class Program
    {
        public static void Main(string[] args)
        {
            LunchQueue lQ = new LunchQueue();
            lQ.TheLunchQueue();
            //TheLunchQueue();
        }

      /*  public static int GetInt(string prompt)
        {
            Console.WriteLine(prompt);
            int num = int.Parse(Console.ReadLine());
            return num;
        }

        public static string GetString(string prompt)
        {
            Console.WriteLine(prompt);
            string val = Console.ReadLine();
            return val;
        }

        public static void TheLunchQueue()
        {
            string mainDish = GetString("What main dish would you like(Fish, Burgers or veg) ?");
            int potatos = GetInt("How many roast potatos would you like?");
            int sprouts = GetInt("How many Brussel sprouts would you like?");
            string output = $"Hello, your lunch is {mainDish} with {potatos} roast potatoes and {sprouts} Brussel sprouts";
            Console.WriteLine(output);
        }*/

    }

    class LunchQueue{
        public int GetInt(string prompt)
        {
            Console.WriteLine(prompt);
            int num = int.Parse(Console.ReadLine());
            return num;
        }

        public string GetString(string prompt)
        {
            Console.WriteLine(prompt);
            string val = Console.ReadLine();
            return val;
        }

        public void TheLunchQueue()
        {
            string mainDish = GetString("What main dish would you like(Fish, Burgers or veg) ?");
            int potatos = GetInt("How many roast potatos would you like?");
            int sprouts = GetInt("How many Brussel sprouts would you like?");
            string output = $"Hello, your lunch is {mainDish} with {potatos} roast potatoes and {sprouts} Brussel sprouts";
            Console.WriteLine(output);
        }

    }

}